### PR TITLE
Add more TS notification sounds

### PR DESCRIPTION
**Pull request by sinf**
Added some audio listed here: https://code.google.com/p/xcc/source/browse/trunk/xcc/data/ts%20mix%20description.txt

### CODE REVIEW

**Comment by Mailaender** [comment]
:+1: Thanks!

**Comment by reaperrr** [comment]
I appreciate the effort, but I don't think this should be merged in its current state.

1) Not a major problem, but there seem to be a few typos.

2) Most of the additions under sounds related to explosions, weapons and some other effects are only useful if you want to trigger a specific sound in a map without using a weapon/explosion, otherwise they're completely superflous because weapons, explosions etc. call the ```.aud``` files directly and don't use notifications.

3) Another potential problem is that this contains all Firestorm speeches and sounds. To my knowledge, we're concentrating on vanilla TS initially, and if our content download/installation doesn't take this into consideration, it might result in crashes if expand01.mix and its content are missing and some mapper tries to add a sound because we had it defined in notifications.yaml. Additionally, this bloats the yaml with sound and speech definitions we currently don't need.

4) This is simply too much for a single yaml, in my opinion. I'd at least split into speech and sounds, and then sub-split between ```generic and ```singleplayer```, since - as mentioned above - most of these sounds and speeches will only be used in map triggers (if at all), and I dare to say those won't play much of a role for quite some time to come.


To be perfectly honest, I'd prefer to add notifications only in moderation and when needed over this "slap everything in" approach. In my opinion, such a monster yaml would actually make things harder for us as long as the mod is still under development.

**Comment by penev92** [comment]
I agree with the first part of @reaperrr's comment. Explosions and other sound effects have no place in `notifications.yaml`, and our TS mod should not contain any FS assets.
That being said, I don't agree with the last part - not "registering" assets in mod YAMLs has lead to enough headaches in TD, RA and D2k. We should have everything available for when we need it, not make people have to go through the mix files searching for the 1 sound they need every time they need something new.
All TS voice notifications should be added here. Nothing more, nothing less. Splitting into separate YAML files might be good, so once we see how many all notifications are we can discuss that.

**Comment by sinf** [comment]
I removed all speech notifications not present in either speech01.mix or speech02.mix (only vanilla TS speech remain) and also removed explosion/weapon/misc effects.

File: mods/ts/audio/speech-generic.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		Repairing: 00-i040
+		SecondaryObjectiveAchieved: 00-i102
+		SelectTarget: 00-i042
+		SilosNeeded: 00-i044
+		SoundTheAlarm: 41-n002
+		SpyKilledMissionFailed: 01-n284
+		SpyLostMissionFailed: 01-n210
+		StartGame: 00-i200
+		StructureSold: 00-i228
+		SubterrainianUnitDetected: 00-i174
```
Comment (reaperrr) [inline mods/ts/audio/speech-generic.yaml]: Typos, should be 'Subterranean'

File: mods/ts/audio/speech-generic.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		ConstructionComplete: 00-i018
+		CriticalStructureLost: 00-i196
+		CriticalUnitLost: 00-i194
+		EmPulseCannonReady: 00-i158
+		EstablishingBattleControllsStandBy: 01-n006
+		FireStormDefenseOffline: 00-i170
+		FireStormDefenseReady: 00-i162
+		FourMinutesRemaining: 00-i128
+		HarvesterAttack: 00-i090
+		InsufficantFunds: 00-i022
```
Comment (reaperrr) [inline mods/ts/audio/speech-generic.yaml]: Insufficient

File: mods/ts/audio/speech-generic.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+		Cancelled: 00-i220
+		ChemicalMissileReady: 00-i152
+		CivillianKilled: 00-n018
+		CloakedUnitDetected: 00-i172
+		ClusterMissileReady: 00-i154
+		ConstructionComplete: 00-i018
+		CriticalStructureLost: 00-i196
+		CriticalUnitLost: 00-i194
+		EmPulseCannonReady: 00-i158
+		EstablishingBattleControllsStandBy: 01-n006
```
Comment (reaperrr) [inline mods/ts/audio/speech-generic.yaml]: Controls with only one 'l'.

File: mods/ts/audio/speech-generic.yaml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+		BattleControllsEstablished: 01-n008
+		BridgeRepaired2: 00-n128
+		BridgeRepaired: 00-i118
+		Building: 00-i216
+		BuildingCannotPlaceAudio: 00-i016 #CannotDeployHere
+		BuildingCaptured: 00-i056
+		BuildingInfiltrated: 00-i014
+		Cancelled: 00-i220
+		ChemicalMissileReady: 00-i152
+		CivillianKilled: 00-n018
```
Comment (reaperrr) [inline mods/ts/audio/speech-generic.yaml]: Civilian (one l too much).

File: mods/ts/audio/speech-generic.yaml [hunk 5]
```diff
@@ -0,0 +1,81 @@
+Speech:
+	Notifications:
+		AirUnitLost: 00-i074
+		AirstrikeReady: 00-n160
+		AllianceBroken: 00-i306
+		AllianceFormed: 00-i304
+		BaseAttack: 00-i082
+		BaseDefensesOffline: 00-i180
+		BasePerimiterHasBeenBreached: 01-n005
```
Comment (reaperrr) [inline mods/ts/audio/speech-generic.yaml]: Perimeter

File: mods/ts/audio/speech-generic.yaml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+		NewOptions: 00-i032
+		NewTerrainDiscovered: 00-i198
+		NoBuild: 00-i064
+		OnHold: 00-i218
+		OurAllyIsUnderAttack: 00-i308
+		PlayerHasResigned: 00-i252
+		PlayerWasDefeated: 00-i268
+		PrimaryBuildingSelected: 00-i034
+		PrimaryObjectiveAchieved: 00-i100
+		ProductionFacillityDestroyed: 01-n233
```
Comment (reaperrr) [inline mods/ts/audio/speech-generic.yaml]: Facility

File: mods/ts/audio/speech-generic.yaml [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+		QuaterneryObjectiveAchieved: 00-i106
```
Comment (reaperrr) [inline mods/ts/audio/speech-generic.yaml]: Quarternary

File: mods/ts/audio/speech-singleplayer.yaml [hunk 8]
```diff
@@ -0,0 +1,228 @@
+Speech:
+	Notifications:
+		AirPowerInnefectiveDuringIonStorms: 00-n072
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Ineffective

File: mods/ts/audio/speech-generic.yaml (hunk 3 above)
Comment (penev92) [inline mods/ts/audio/speech-generic.yaml]: Shouldn't that be "Control" as well?

File: mods/ts/audio/speech-singleplayer.yaml [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
+		BuildBarracksToTrainAdditionalTroops: 00-i312
+		BuildHandOfNodToTrainAdditionalTroops: 00-i314
+		BuildMorePowerPlantsToRestoreFullPower: 00-i310
+		BuildSilosToStoreExcessTiberium: 00-i318
+		BulletTrainDeparting: 01-n074
+		C4PlantedMissionComplete: 00-n180
+		CaptureNodTechCenterToRecoverTheCrystals: 00-n134
+		CaptureTheTvStationToTheEast: 01-n020
+		CapturingTheLocalRadarTowers: 01-n038
+		CivillianCityIsUnderAttack: 00-n255
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Civilian

File: mods/ts/audio/speech-singleplayer.yaml [hunk 10]
```diff
@@ -0,0 +1,228 @@
...
+		CivilliansEvacuatedObjectiveComplete: 00-n024
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Civilians

File: mods/ts/audio/speech-singleplayer.yaml [hunk 11]
```diff
@@ -0,0 +1,228 @@
...
+		DestroyAllNodForcesInTheArea: 00-n006
+		DestroyHassansEliteGuard: 01-n002
+		DestroyHassansPyramid: 01-n034
+		DestroyHassansRemainingGuard: 01-n024
+		DoNotAllowTheConstructionYardToBeDestroyed: 01-n178
+		DoNotLetMcnealEscape: 01-n229
+		DontShootICanTellYouGdisEvacLocations: 47-n000
+		EnemySamSitesDetected: 00-n084
+		EtaForAirstrikeIs5Minutes: 35-n004
+		EyeOfTheStormHasBeenEnteredEqupimentAtMaximumEfficency: 00-n241
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Efficiency

File: mods/ts/audio/speech-singleplayer.yaml [hunk 12]
```diff
@@ -0,0 +1,228 @@
...
+		FighterProductionFacillityDestroyedMissionComplete: 00-n210
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Facility

File: mods/ts/audio/speech-singleplayer.yaml [hunk 13]
```diff
@@ -0,0 +1,228 @@
...
+		FighterProductionFacillityLocatedDropshipsEnRoute: 00-n208
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Same here

File: mods/ts/audio/speech-singleplayer.yaml [hunk 14]
```diff
@@ -0,0 +1,228 @@
...
+		GetTratosOverHereAsap: 35-n000
+		GhostStalkerTerminatedMissionFailed: 00-n182
+		GladToSeeYouFreeSir: 38-n024
+		HarvestTheTiberiumToTheNorth: 01-n000
+		HarvesterWillBeginHarvesting: 00-n002
+		HassanHasBeenCaptured: 38-n012
+		HassanIsEscaping: 38-n010
+		HereComesKanesWelcomingComittee: 33-n022
+		HereTheyCome: 33-n008
+		IAmFreeNowItsPabackTimeHassan: 39-n000
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Pa'y'back

File: mods/ts/audio/speech-singleplayer.yaml [hunk 15]
```diff
@@ -0,0 +1,228 @@
...
+		IncommingTransmission2: 00-n056
+		IncommingTransmission: 00-i020
+		InferiorTacticsDetected2: 01-i342
+		InferiorTacticsDetected: 00-i344
+		IntrudersDetected: 01-n320
+		IonCannonFiringResendingControllCodes: 01-n268
+		IonStormFormingOverThisSectorPilotReturnToBase: 00-n140
+		IonStornWillDisableFighters: 00-n192
+		IsThereAnyTechLeftAtTheNodBase: 33-n010
+		ItSeamsThatCretainStructuresAttractLighting: 01-n054
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Seems

File: mods/ts/audio/speech-singleplayer.yaml [hunk 16]
```diff
@@ -0,0 +1,228 @@
...
+		ItWouldBeAdvantagesToDestroyBridges: 00-n042
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Advantageous

File: mods/ts/audio/speech-singleplayer.yaml [hunk 17]
```diff
@@ -0,0 +1,228 @@
...
+		ItsNodSirTheyCameOutOfNowhere: 33-n002
+		KillAllPrisoners: 01-n324
+		KodiakDestroyedMissionFailed: 00-n240
+		KodiakInCriticalCondition: 00-n239
+		KodiakUnderAttack: 00-n236
+		LaserTurretsRunForIt: 38-n006
+		LetsGetHimToTheConvoyPoint: 38-n020
+		LooksLikeTheyToreThisPlaceAppartAlready: 33-n004
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Apart

File: mods/ts/audio/speech-singleplayer.yaml [hunk 18]
```diff
@@ -0,0 +1,228 @@
...
+		MainPowerOffline: 01-n328
+		MainPowerRestored: 01-n330
+		McnealCapturedMissionComplete: 01-n232
+		McnealEscapedMissionFailed: 01-n234
+		McnealKilledMissionFailed: 01-n230
+		McvHasArrivedToTheSouthEast: 01-n040
+		MetiorStormApproaching: 00-i178
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Meteor

File: mods/ts/audio/speech-singleplayer.yaml [hunk 19]
```diff
@@ -0,0 +1,228 @@
...
+		MutantCommandosAvailable: 00-i290
+		MutantLifeFormDetectedOnBoard: 01-n116
+		MutantSuppliesFound: 00-i208
+		MutantVerminDetected: 01-n070
+		MutantsDetectedEnemyForcesEnRoute: 00-n206
+		MutantsEnRouteToPowerGrid: 00-n168
+		MutantsLocated: 01-n128
+		MutantsLostMissionFailed: 00-n158
+		NegativeICanMakeIt: 35-n010
+		NodHasInstalledSeveralIcbmsToTakeOutThePhilladelphia: 00-n252
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Philadelphia

File: mods/ts/audio/speech-singleplayer.yaml [hunk 20]
```diff
@@ -0,0 +1,228 @@
...
+		NodSamSitesDestroyed: 00-n020
+		ObserveSuperiorTacticsWhileYouStillHaveHumanEyes: 01-i350
+		OneMinuteRemaining: 00-i134
+		Orbit1Complete: 01-n256
+		Orbit2Complete: 01-n258
+		Orbit3CompleteMissionFailed: 01-n260
+		OurWorkHereIsDoneWereOutOfHere: 10-n044
+		PerimeterDeactivated: 00-n112
+		PerimiterDeactivatedCommenceAttack: 01-n262
+		PhilladelphiaInRangeIcbmLaunchDetectedMissionFailed: 00-n248
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Philadelphia

File: mods/ts/audio/speech-singleplayer.yaml [hunk 21]
```diff
@@ -0,0 +1,228 @@
...
+		PickupService: 35-n008
+		PilotReturnToBase: 00-n142
+		PowerLevelsAreLowConstructMorePowerPlants: 01-n010
+		PrepairForDecimation: 01-i362
+		PreventGdisEvacuationAtAllCosts: 01-n102
+		PreventThatTrainFromLeaving: 01-n076
+		ProbobleObjectiveIsRescueOfMutantPrisoners: 01-n322
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Probable

File: mods/ts/audio/speech-singleplayer.yaml [hunk 22]
```diff
@@ -0,0 +1,228 @@
...
+		ProceedToEvacLocationTransportEnRoute: 01-n286
+		ProtectYourEngineers: 01-n086
+		ReenteringIonStormCautionIsAdvised: 00-n245
+		ReinforcementsArrived: 00-i038
+		ReinforcementsEnRoute: 00-n132
+		ReinforcementsInbound: 00-n114
+		ResearchFacillityDestroyed: 01-n134
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Facility

File: mods/ts/audio/speech-singleplayer.yaml [hunk 23]
```diff
@@ -0,0 +1,228 @@
...
+		ResearchFacillityLocated: 01-n132
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Same here

File: mods/ts/audio/speech-singleplayer.yaml [hunk 24]
```diff
@@ -0,0 +1,228 @@
...
+		RetreatIsYourOnlyLogicalOption: 00-i346
+		SamSitesDestroyedDropshipsInbound: 00-n088
+		SiteSecureObjectiveComplete: 00-n032
+		SoMuchForSutlty: 10-n038
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Subtlety

File: mods/ts/audio/speech-singleplayer.yaml [hunk 25]
```diff
@@ -0,0 +1,228 @@
...
+		SomethingsWrongWeShouldHaveBeenThereByNow: 36-n004
+		StandAndIdentifyYourselfInTheNameOfKane: 41-n000
+		StealthIsKey: 01-n208
+		StormAbatingCommenceAttackOnNodForces: 00-n238
+		SupplyBaseDestroyedMissionComplete: 00-n166
+		SurrenderIsPreferableAgainstTheOddsYouAreFacing: 00-i372
+		SurrenderIsYourOnlyLogicalOption: 00-i348
+		TakePointSoldier: 38-n022
+		TascitusHasBeenAquired: 01-n066
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Tacitus

File: mods/ts/audio/speech-singleplayer.yaml [hunk 26]
```diff
@@ -0,0 +1,228 @@
...
+		TechnologyCentreCapturedObjectiveComplete: 00-n034
+		TenMinutesRemaining: 00-i124
+		TerminatorProtocolsInitiated: 01-i378
+		ThanksForYourHelp: 37-n000
+		ThanksWellEnjoyThis: 36-n002
+		ThatTrainGoesToThePowerGridWeToldYouAboutGetUsOnBoard: 36-n000
+		TheCavelryHasArrived: 33-n014
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Cavalry

File: mods/ts/audio/speech-singleplayer.yaml [hunk 27]
```diff
@@ -0,0 +1,228 @@
...
+		TheCreatureIsTheTiberiumSubstanceWeSeekUseIt: 01-n174
+		TheIceWontSupportTheWeightOfMultipleVechiclesCrossing: 38-n028
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Vehicles

File: mods/ts/audio/speech-singleplayer.yaml (hunk 15 above)
Comment (penev92) [inline mods/ts/audio/speech-singleplayer.yaml]: Certain

File: mods/ts/audio/speech-singleplayer.yaml [hunk 28]
```diff
@@ -0,0 +1,228 @@
...
+		TheIonCannonIsOurs: 01-n270
+		TheMedicalFacillityIsLocatedToTheEast: 10-n032
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Facility

File: mods/ts/audio/speech-singleplayer.yaml [hunk 29]
```diff
@@ -0,0 +1,228 @@
...
+		TheMedicalFacillityIsLocatedToTheWest: 10-n034
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Same here

File: mods/ts/audio/speech-singleplayer.yaml [hunk 30]
```diff
@@ -0,0 +1,228 @@
...
+		TheMutantsHaveTurnedOnUs: 38-n016
+		TheSacrificeOfTheManyIsThePleasureOfTheFew: 01-i368
+		TheTraitorsAreCommingBlowTheBridge: 38-n008
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Coming

Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Coming

File: mods/ts/audio/speech-singleplayer.yaml [hunk 31]
```diff
@@ -0,0 +1,228 @@
...
+		ThereEverywhere: 38-n018
+		TheseCrittersDontLookTooFriendly: 33-n012
+		TheyMayBeOnToUs: 37-n002
+		TheyveSeenUsFallBack: 10-n046
+		ThreeMinutesRemaining: 00-i130
+		TiberiamIsLethalToUnprotectedInfantry: 38-n002
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Tiberium

File: mods/ts/audio/speech-singleplayer.yaml [hunk 32]
```diff
@@ -0,0 +1,228 @@
...
+		TiberiumIsHazzardusToUnprotectedInfantry: 01-n009
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Hazardous

File: mods/ts/audio/speech-singleplayer.yaml [hunk 33]
```diff
@@ -0,0 +1,228 @@
...
+		TiberiumLifeFormDetected2: 00-n249
+		TiberiumLifeFormDetected3: 01-n068
+		TiberiumLifeFormDetected: 00-n243
+		TiberiumMissileLaunchedMissionFailed: 00-n250
+		TimeToEraseTheHumanFactorFromThisEquasion2: 01-i360
+		TimeToEraseTheHumanFactorFromThisEquasion: 00-i360
+		ToBuildOrTrain: 01-n007
+		ToCaptureABuilding: 01-n026
+		ToDeployAVechile: 01-n027
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Vehicle

File: mods/ts/audio/speech-singleplayer.yaml [hunk 34]
```diff
@@ -0,0 +1,228 @@
...
+		UfoDestroyedMissionFailed: 00-n054
+		UfoUnderAttack: 00-n052
+		UseThemToYourAdvantage: 01-n055
+		WarningRegulatorsOffline: 01-n900
+		WeAreHereWaitingForYourArrival: 44-n000
+		WeGotAnOldStockPileHereSir: 38-n014
+		WeHaveBeenBetrayedGdiKidnappedTratos: 36-n008
+		WeHaveBeenTouchedByTheSpiritHandOfKane: 40-n000
+		WeJustGotWordFromGeneralHassanHesMovingUpNorthToCairo: 41-n006
+		WeMustGetTratosOutOfThatFacillity: 10-n036
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Facility

File: mods/ts/audio/speech-singleplayer.yaml [hunk 35]
```diff
@@ -0,0 +1,228 @@
...
+		WeMustGetTratosToTheTransport: 10-n042
+		WeNeedMoreMenTrainThemFromTheHandOfNod: 38-n004
+		WeTireOfYourInsignificantDefience: 01-i366
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Defiance

File: mods/ts/audio/speech-singleplayer.yaml [hunk 36]
```diff
@@ -0,0 +1,228 @@
...
+		WhereTheHellAreThoseReinforcements: 33-n000
+		WithAllDueRespectMoveYourAss: 35-n002
+		YouHaveBeenMarkedForTermination2: 01-i356
+		YouHaveBeenMarkedForTermination: 00-i356
+		YouMakeThisEasyFleshbag: 01-i364
+		YouMustBuildATiberiumWasteFacility: 01-n180
+		YourDefeatIsAtHand2: 01-i352
+		YourDefeatIsAtHand: 00-i352
+		YourForcesHaveBeenDetected: 01-n100
+		YourProbobilityOfSucessIsDropping: 00-i370
```
Comment (reaperrr) [inline mods/ts/audio/speech-singleplayer.yaml]: Probability

**Comment by reaperrr** [comment]
I think you could actually rename sounds-singleplayer.yaml to sounds-generic.yaml, as most of those sounds already are or probably will be used for UI/multiplayer as well.

**Pull request merged**